doc/api/resources/quotas.rst

```diff
@@ -106,6 +106,132 @@ Endpoints
    :statuscode 401: Authentication failure
    :statuscode 403: The requested organizer/event does not exist **or** you have no permission to view this resource.
 
+.. http:post:: /api/v1/organizers/(organizer)/events/(event)/quotas/
+
+   Creates a new quota
+
+   **Example request**:
+
+   .. sourcecode:: http
+
+      POST /api/v1/organizers/bigevents/events/sampleconf/quotas/1/ HTTP/1.1
```

Comment:
this is only to ``/quotas/``, not ``/quotas/1/``, I guess :wink:

Reply:
Correct - and body shouldn't contain `"id": 1,` - I'll have a word with the copy-paste monkey 🙈